.travis.yml

```diff
@@ -2,20 +2,21 @@ language: python
 python:
   - "3.4"
   - "3.5"
-#  - "3.5-dev" # 3.5 development branch
-#  - "nightly" # currently points to 3.6-dev
+  - "3.6"
 # command to install dependencies
 install: 
-    - pip install -r requirements.txt
-    - pip install -r requirements-dev.txt
-    - pip install -r requirements-docs.txt
+  - pip install -r requirements.txt -r requirements-dev.txt
```

Comment:
_requirements-dev.txt_ should be enough for a developer (or CI) to run the tests.